.github/workflows/build-lint-test.yml

```diff
@@ -24,19 +24,27 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
 
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 'latest'
+
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
           node-version: '18'
```

Comment:
remember to add the pnpm cache here too

Reply:
Added the cache
[d9dc313](https://github.com/upleveled/preflight/pull/379/commits/d9dc313a2cb7967a7968f8f11391188e3f37d160)